ci-operator/step-registry/multiarch/mto/deploy-operand/OWNERS

```diff
@@ -0,0 +1,6 @@
+approvers:
```

Comment:
can we rename the `mto` folder to `tuning-operator`, and have `deploy/` for the operator, `deploy/pod-placement` for the pod-placement operand?